server/server/data_loader/po.py

```diff
@@ -205,6 +203,22 @@ def load_po_texts(change_tracker, po_dir, db):
 
 
 def generate_html(markup: str, translation: List[List[str]]) -> str:
+    """ Takes HTML markup and insert translated string between sc-seg tags with id matching text uid.
```

Comment:
https://www.python.org/dev/peps/pep-0257/

`The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".`

Reply:
You have all that information here. It is Google's style doc-string

Reply:
You use both a descriptive style (`Takes`) and command style (`insert`) here. Is that Google style? :)